server/routes/ads.js

```diff
@@ -39,7 +39,7 @@ router.get('/', async (req, res, next) => {
   let result = await processData(getRes.data.values);
 
   if (!id) {
-    res.status(200).json(result);
+    res.status(200).json({ data: { ads: result } });
```

Comment:
non-blocking comment. changing shape of existing response is inconsistent with rest of route responses. since we are going away with this route, it doesn't matter at this point